frontend/src/components/NotFound.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import error from "../assets/error.jpg";
+
+
+const NotFound = () => {
+    return (
+        <>
+            <section>
+                <div className="form-container">
+                    <h1 className="header">Page not found</h1>
+                    <Link to="/">
+                        <div>
+                            <img className="error-img" src={error} alt="error" />
```

Comment:
This was also awesome - meaning that we have to do some garden work as a punish if we do any mistake in the website :-)